packages/regl-worldview/src/Worldview.js

```diff
@@ -187,15 +186,24 @@ export class WorldviewBase extends React.Component<BaseProps, State> {
       return;
     }
 
-    if (!readHitmap) {
-      return handleMouseInteraction(0, ray, e, handler);
+    if (!readHitmap && worldviewHandler) {
+      return handleWorldviewMouseInteraction(0, ray, e, worldviewHandler);
     }
 
     // reading hitmap is async so we need to persist the event to use later in the event handler
     (e: any).persist();
     worldviewContext
       .readHitmap(canvasX, canvasY)
-      .then((objectId) => handleMouseInteraction(objectId, ray, e, handler))
+      .then((objectId) => {
+        if (worldviewHandler) {
+          handleWorldviewMouseInteraction(objectId, ray, e, worldviewHandler);
+        }
+        const componentLevelHandler = worldviewContext.mouseHandlers[handlerName][objectId];
+        if (componentLevelHandler) {
+          const [component, drawProp] = componentLevelHandler;
+          component.props[handlerName](objectId, ray, e, drawProp);
+        }
```

Comment:
so this means if the component has a mouse handler and worldview also has a handler for the same event they'll both fire.  Should we support `e.stopPropagation` in the component level handler?

Reply:
hmm, doesn't seem straightforward to listen into an event to see if `stopPropagation` was called. We'd have to add our own state to each event if we want to do something similar. How often do ya think this use case will happen?

Reply:
It looks like you can't read if `e.stopPropagation` has been called on an event.  I think we can add it later if we want to figure out a way at that point.